src/patternfly/components/DualListSelector/dual-list-selector-header.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-dual-list-selector__header{{#if dual-list-selector-header--modifier}} {{dual-list-selector-header--modifier}}{{/if}}"
```

Comment:
`__header` in case we add something like a description, close button, etc (more than just the title).

`__title` to establish the title section, with the `__title-text` to designate the text - which allows us to add an icon or action or something else in `__title` that's adjacent to `__title-text`.

Seems like a repeatable structure we have in a lot of components.